conda-forge.yml

```diff
@@ -2,5 +2,3 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
```

Comment:
These are recommended and won't affect the build here.  I suggest you to restore those lines.